community/modules/compute/schedmd-slurm-gcp-v6-nodeset/variables.tf

```diff
@@ -29,6 +29,10 @@ variable "node_conf" {
   description = "Map of Slurm node line configuration."
   type        = map(any)
   default     = {}
+  validation {
+    condition     = lookup(var.node_conf, "Sockets", null) == null
+    error_message = "DEPRECATED: With the use of Boards in V6 the Sockets field is deprecated, use SocketsPerBoard instead."
```

Comment:
I would avoid words `DEPRECATED` and `V6`. Let's word it somewhat likes this: 
```
`Sockets` settings conflicts with `SocketsPerBoard` that is automatically computed by SlurmGCP.
Instead you can override `Boards`, `SocketsPerBoard`, `CoresPerSocket`, and `ThreadsPerCore`.
See: https://slurm.schedmd.com/slurm.conf.html#OPT_Boards
```

Reply:
That message seems a lot more descriptive, only change I made was to add slurm link about sockets for additional context on sockets and socketsperboard being mutually exclusive params, etc.